jolt-core/src/dag/stage.rs

```diff
@@ -0,0 +1,94 @@
+use crate::dag::state_manager::StateManager;
+use crate::field::JoltField;
+use crate::poly::commitment::commitment_scheme::CommitmentScheme;
+use crate::subprotocols::sumcheck::BatchableSumcheckInstance;
+use crate::utils::transcript::Transcript;
+
+pub trait SumcheckStages<
+    F: JoltField,
+    ProofTranscript: Transcript,
+    PCS: CommitmentScheme<ProofTranscript, Field = F>,
+>: Send + Sync
+{
+    // Stage 1 is special case of outer sumchec from spartan
+    fn stage1_prove(
+        &self,
+        state_manager: &mut StateManager<'_, F, ProofTranscript, PCS>,
+    ) -> Result<(), anyhow::Error> {
+        let _ = state_manager;
+        Ok(())
+    }
```

Comment:
Is this just to appease clippy? if so prefer to do this
```suggestion
    fn stage1_prove(
        &self,
        _state_manager: &mut StateManager<'_, F, ProofTranscript, PCS>,
    ) -> Result<(), anyhow::Error> {
        Ok(())
    }
```